src/main/game/Board.java

```diff
@@ -3,78 +3,67 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static java.util.Arrays.asList;
-import static main.game.Marks.O;
-import static main.game.Marks.X;
+import static main.game.Mark.*;
 import static main.game.Result.TIE;
 
 public class Board {
 
-    public final int ROW_COUNT = 3;
-    public List<String> grid;
+    private int size;
+    public List<Mark> grid;
 
-    public Board(List<String> grid) {
+    public Board(int size, List<Mark> grid) {
```

Comment:
Why do you need this second constructor?
Also, isn't the length of the `List<Mark>` equal to the `size` parameter?